src/core/function_elt.tf

```diff
@@ -78,6 +93,20 @@ module "function_elt" {
 
     PROFILE_TOPIC_NAME              = "io-cosmosdb-profiles"
     PROFILE_TOPIC_CONNECTION_STRING = module.event_hub.keys["io-cosmosdb-profiles.io-fn-elt"].primary_connection_string
+
+    COSMOSDB_REPLICA_NAME = "db"
+    COSMOSDB_REPLICA_URI  = replace(data.azurerm_cosmosdb_account.cosmos_api.endpoint, "io-p-cosmos-api", "io-p-cosmos-api-northeurope")
```

Comment:
```suggestion
    COSMOSDB_REPLICA_URI  = "${data.azurerm_cosmosdb_account.cosmos_api.endpoint}-northeurope"
```

Reply:
Not sure it's gonna work this way
I'm replacing those strings into an url like `https://<name>.documents.azure.com:443/`

Reply:
ok u're right